toolkit/tools/pkg/imagecustomizerlib/testdata/overlays-config.yaml

```diff
@@ -36,21 +38,35 @@ storage:
     mountPoint:
       path: /
 
+  - deviceId: var
+    type: ext4
+    mountPoint:
+      path: /var
+      options: defaults,x-initrd.mount
+
 os:
   resetBootLoaderType: hard-reset
 
   packages:
     install:
-    - dracut-overlayfs
     - openssh-server
-    - vim
 
-  additionalFiles:
-    # Workaround until updated dracut-overlayfs package has been published.
-    ../../../../../SPECS/dracut/20overlayfs/module-setup.sh: /lib/dracut/modules.d/20overlayfs/module-setup.sh
-    ../../../../../SPECS/dracut/20overlayfs/overlayfs-mount.sh: /lib/dracut/modules.d/20overlayfs/overlayfs-mount.sh
+  services:
+    enable:
+    - sshd
 
   overlays:
```

Comment:
Should this include some of the initrd mounts and dependencies as well?

Reply:
If overlay is rootfs overlay, then users will enable `isRootfsOverlay` to `True`, at the same time, the mountDependencies will be available during the initrd phase with the extra mount option `x-initrd.mount`. We have more details in the README file.